resteasy-genericresponse/src/main/java/com/cerner/beadledom/resteasy/BuiltGenericResponse.java

```diff
@@ -255,4 +256,15 @@ public Link getLink(String relation) {
   public String getHeaderString(String name) {
     return rawResponse.getHeaderString(name);
   }
+
+  @Override
+  public String toString() {
+    return "BuiltGenericResponse{body=" + body + ", bodyClass=" + bodyClass + ", errorBody=" + errorBody
+            + ", rawResponse=Response{status=" + rawResponse.getStatus() + ", mediaType="
```

Comment:
Rather than duplicating the `rawResponse` String code, it might be better to have a class like `ResponseToStringWrapper` that only has a `toString` method on it.